recipes/r-sodium/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = '1.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sodium
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sodium_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sodium/sodium_{{ version }}.tar.gz
+  sha256: dcb855af1fe66140885692eb4cb51ab8b1f02d028efa8c4b630fdabe3b8abfff
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{native}}toolchain        # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}sed               # [win]
+    - {{posix}}grep              # [win]
+    - {{posix}}autoconf
+    - {{posix}}automake          # [not win]
+    - {{posix}}automake-wrapper  # [win]
+    - {{posix}}pkg-config
+    - {{posix}}make
+    - {{posix}}sed  # [win]
```

Comment:
sed and coreutils are duplicated

Reply:
Good catch! Looks like that's a bug in the conda_r_skeleton_helper.
Anyway, looks like it's passed tests again.